x-pack/metricbeat/packages.yml

```diff
@@ -0,0 +1,71 @@
+specs:
```

Comment:
`dev-tools/packaging/packages.yml` also contains instructions for OSS and non-OSS packages, why is it needed here too?

If needed we should also add the docker type.